app/views/layouts/mailer.html.haml

```diff
@@ -8,16 +8,16 @@
     = stylesheet_link_tag 'mail'
   %body{:bgcolor => "#FFFFFF" }
     - unless @hide_ofn_navigation
-      %table.head-wrap{:bgcolor => "#f2f2f2"}
+      %table.head-wrap
```

Comment:
I wonder if the colour was set here for email compatibility reasons
_googles_...

Nope, all good, CSS background-color should be just as compatible as bgcolor:
https://www.litmus.com/blog/background-colors-html-email#how-to-code-solid-color

That article also notes that:
> The safest place to add the background color is on a table or a table cell.

So we can't assume that the email will have a white background (for example if the user's device is in dark mode.) I think that's fine, although it would be interesting to test it out to see what it's like.